taxcalc/tests/test_calculate.py

```diff
@@ -117,14 +118,21 @@ def test_make_Calculator_deepcopy():
 
 
 def test_make_Calculator_from_files(paramsfile):
-    calc = Calculator.from_files(paramsfile.name, tax_dta_path,
-                                 start_year=1991,
-                                 inflation_rates=irates)
+    with open(paramsfile.name) as pfile:
+        params = json.load(pfile)
+    ppo = Parameters(parameter_dict=params, start_year=1991,
```

Comment:
Here is a case where it seems that we have "handicapped" the Parameters class. Why not permit:

```
ppo = Parameters("myparams.json", start_year=1991, num_years=10)
```
or something similar to
```
ppo = Parameters(path="myparams.json", start_year=1991, num_years=10)
```

It seems particularly useful to have such a feature in light of the upcoming feature https://github.com/OpenSourcePolicyCenter/Tax-Calculator/issues/284, which we hope to integrate to the webapp. This would allow a non-coder to create a policy reform, download a JSON file, give that to someone who could use `taxcalc` in a notebook, and then the Python user could create the individual's reform with something like:

```
ppo = Parameters(path="bobs_reform.json", start_year=1991, num_years=10)
```

which seems nice.